modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -717,7 +731,7 @@ RETURN d.timezone, d.offset, d.offsetMinutes, d.epochSeconds, d.epochMillis
 .+date.truncate+
 ======
 
-Get the first _Date_ of the current year:
+Get the first date of the current year:
```

Comment:
I suppose this should say "first _day_", as also the return name? First _dates_ are usually much more romantic than our docs.

Reply:
```suggestion
Get the first day of the current year:
```